src/redturtle/prenotazioni/adapters/prenotazione_email_message.py

```diff
@@ -215,8 +214,13 @@ def message_subject(self) -> str:
         return translate(
             _(
                 "new_booking_admin_notify_subject",
-                default="New booking for ${context}",
-                mapping={"context": booking_folder.title},
+                default="[${context} - gate ${gate}] New booking on ${date} by ${name}",
```

Comment:
hai verificato con degli esempi reali che non sia troppo lungo l'oggetto così in un client di posta?